charts/enterprise/authelia/questions.yaml

```diff
@@ -584,13 +602,13 @@ questions:
                       label: "Host"
                       schema:
                         type: string
-                        default: "smtp.mail.svc.cluster.local"
+                        default: "smtp.gmail.com"
```

Comment:
```suggestion
                        default: "smtp.mail.svc.cluster.local"
```

Reply:
Steven was just the uid the guy that made that guide used.

But having real domains as placeholders is bad idea, there is a domain that anyone can safely use.
`example.com`